style.css

```diff
@@ -515,3 +948,479 @@ body {
   }
 }
 
+@media (min-width: 992px) {
+  .middle-section__clutch {
+    padding: 0 61px 0 55px;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+    min-height: 556px;
+    margin-top: 14px;
+  }
+    
+  .middle-section__title {
+    margin-top: 0;
+    margin-bottom: 16px;
+    padding-top: 44px;
+  }
+
+  .middle-section__subtitle {
+    margin-top: 0;
+    margin-bottom: 60px;
+  }
+
+  .middle-section__desription {
+    margin-top: 0;
+    margin-bottom: 43px;
+    opacity: 0.5;
+  }
+
+  .middle-section__btn_other {
+    position: absolute;
+    right: 50px;
+    top: 95%;
+    width: 478px;
+    cursor: pointer;
+    text-decoration: none;
+  }
+
+  .middle-section__btn_busket {
+    width: 253px;
+    height: 44px;
+    padding-right: 10px;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+  }
+
+  .middle-section__colors-and-price {
+    min-width: 360px;
+    margin-top: 0;
+    margin-bottom: 54px;
+  }
+
+  .middle-section__form-colors {
+    width: 161px;
+    height: 39px;
+  }
+
+  .form__colors-item::before {
+    left: 0;
+    width: 30px;
+    height: 30px;
+  }
+
+  input[type="radio"]:checked + .form__colors-item::before {
+    width: 38px;
+    height: 38px;
+    -webkit-transform: translate(-4px,-9%);
+    -ms-transform: translate(-4px,-9%);
+    transform: translate(-4px,-9%);
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)